sdk-ext-lib/src/main/java/cash/z/ecc/sdk/fixture/WalletAddressesFixture.kt

```diff
@@ -1,21 +1,29 @@
 package cash.z.ecc.sdk.fixture
 
+import cash.z.ecc.android.sdk.type.UnifiedFullViewingKey
 import cash.z.ecc.sdk.model.WalletAddress
 import cash.z.ecc.sdk.model.WalletAddresses
 
 object WalletAddressesFixture {
     // These fixture values are derived from the secret defined in PersistableWalletFixture
-    const val VIEWING_KEY = "03feaa290589a20f795f302ba03847b0a6c9c2b571d75d80bc4ebb02382d0549da"
+    val VIEWING_KEY = UnifiedFullViewingKey(
```

Comment:
While we're here, we should probably just remove the viewing key since we don't need it for the Secant wallet.